create_full_environment.sh

```diff
@@ -6,24 +6,8 @@ function error () {
     echo $@ 1>&2
 }
 
-# Check OS
```

Comment:
Keep the OS check - just expand it.
There are plenty of other distros we do not support yet.

Reply:
not sure it worth it. too much to expand